README.adoc

```diff
@@ -6,6 +6,7 @@
 
 https://github.com/openshift/odo/releases/latest[image:https://img.shields.io/github/v/release/openshift/odo?style=for-the-badge[GitHub release]]
 image:https://img.shields.io/github/license/openshift/odo?style=for-the-badge[License]
+image:https://api.netlify.com/api/v1/badges/e07867b0-56a4-4905-92a9-a152ceab5f0d/deploy-status[![Netlify Status]
```

Comment:
can you please add link that will point to our netlify side? similarly like it is done for release badge

Reply:
Not sure what you mean. 😕

Reply:
Ah you mean it should be a link to a page on netlify instead of just an image. Weirdly, it was working earlier. Let me check what went wrong.

Reply:
Done.